ompi/mca/io/ompio/io_ompio_request.h

```diff
@@ -50,6 +50,11 @@ struct mca_ompio_request_t {
     mca_ompio_request_type_t                       req_type;
     void                                          *req_data;
     opal_list_item_t                               req_item;
+#if OPAL_CUDA_SUPPORT
+    void                                          *req_tbuf;
+    size_t                                         req_size;
+    opal_convertor_t                          req_convertor;
```

Comment:
I can't figure out why there is a need for a second convertor in addition to the one from the base request ?

Reply:
done